### DISCUSSION TITLE
Tuya PIR sensor Home Assistant 2021.11.0b0

### DISCUSSION THREAD

**Discussion by guidohaesen** [tuya/tuya-home-assistant#639] [Q&A] [answered]
Hello hard working devs and testers,

I installed Home Assistant 2021.11.0b0 today, and specially for the Tuya integration.
I deleted V2 and started over with the build in integration. And i must say, **AWESOME**.

But, there is always a but :-) the PIR sensors doesn’t update. It doesn’t release detected status. The door sensors works as it should.
![integration](https://user-images.githubusercontent.com/11542368/139321346-cfe8422f-644e-4883-882a-9d79d04bf5d7.png)
![Device](https://user-images.githubusercontent.com/11542368/139321360-94c856ec-28e4-42b2-b2da-e810ca179997.png)
![settings](https://user-images.githubusercontent.com/11542368/139321363-45a5f77c-ef47-4983-8e9c-d1013ff499a3.png)

Next test: Tuya Water Cooker :grinning:

**Comment by frenck** [comment, marked as answer]
> But, there is always a but :-) the PIR sensors doesn’t update. It doesn’t release detected status. The door sensors works as it should.

Please report issues with the core integration during the beta with the Home Assistant Core issue tracker (as per beta notes as well). Posting it here, will do nothing.

**Reply by guidohaesen** [reply]
OK. Thanks
